meshnesh/android-level-up
# FT Api endpoints creation #155903085

# What does this PR do?

* Create Java Classes for Resources.
* Creating the Retrofit instance

## How should this be manually tested?

* Clone the repo to your local machine.
* Open the project with Android Studio.
* Check the files on the path: `level_up/service`
* Check the files on the path: `level_up/presenter`

## What are the relevant pivotal tracker stories?

* [#155903085](https://www.pivotaltracker.com/story/show/155903085)
* [#155931587](https://www.pivotaltracker.com/story/show/155931587)
